tools/vs2017_vcvarsall.bat

```diff
@@ -0,0 +1,16 @@
+:: Locates and calls VS 2017's vcvarsall.bat
+
+@echo off
+
+for /f "usebackq delims=" %%i in (`"%~dp0\vswhere.bat" -version 15.0 -property installationPath`) do (
```

Comment:
FYI. We are adding VS2019 support. It would be better if we can make the version configurable here.

Reply:
made some changes so it should be easier to specify the version now. when we need VS2019's vcvarsall.bat, just add a vs2019_vcvarsall.bat.